### ISSUE TITLE
Adjust a11y e2e test failures by impact level

### ISSUE THREAD

**Issue by skoeva** [kubernetes-sigs/headlamp#3207]
We should probably change a11y failures in the e2e tests to warnings rather than errors when the impact is not high enough. The errors with the most critical level of impact should be considered as a test failure, whereas lower impact errors could be considered as warnings to look into.

Related to: #3190 

## Screenshot

![Image](https://github.com/user-attachments/assets/88193d9d-5889-4951-9081-90dd7e8c0e2d)

**Comment by illume** [comment]
I think we are required to fix all of these issues.

Btw. Generally this tool has been shown to find around 20-30% of a11y issues compared to manually tested code.
